migrations/2_deploy_organization.js

```diff
@@ -67,21 +67,17 @@ module.exports = async function(deployer) {
       // Voting parameters and schemes params:
       var voteParametersHash = await AbsoluteVoteInst.getParametersHash(votePrec, NULL_ADDRESS);
 
-      await schemeRegistrarInst.setParameters(voteParametersHash, voteParametersHash, AbsoluteVoteInst.address);
-      var schemeRegisterParams = await schemeRegistrarInst.getParametersHash(voteParametersHash, voteParametersHash, AbsoluteVoteInst.address);
-      await globalConstraintRegistrarInst.setParameters(voteParametersHash, AbsoluteVoteInst.address);
-      var schemeGCRegisterParams = await globalConstraintRegistrarInst.getParametersHash(voteParametersHash, AbsoluteVoteInst.address);
-      await upgradeSchemeInst.setParameters(voteParametersHash, AbsoluteVoteInst.address);
-      var schemeUpgradeParams = await upgradeSchemeInst.getParametersHash(voteParametersHash, AbsoluteVoteInst.address);
+      await schemeRegistrarInst.initialize(AvatarInst.address,AbsoluteVoteInst.address,voteParametersHash, voteParametersHash, );
+      await globalConstraintRegistrarInst.initialize(AvatarInst.address, AbsoluteVoteInst.address, voteParametersHash);
+      await upgradeSchemeInst.initialize(AvatarInst.address, AbsoluteVoteInst.address, voteParametersHash);
 
-      await contributionRewardInst.setParameters(voteParametersHash, AbsoluteVoteInst.address);
-      var contributionRewardParams = await contributionRewardInst.getParametersHash(voteParametersHash, AbsoluteVoteInst.address);
+      await contributionRewardInst.initialize(AvatarInst.address, AbsoluteVoteInst.address, voteParametersHash);
 
       var schemesArray = [schemeRegistrarInst.address,
                           globalConstraintRegistrarInst.address,
                           upgradeSchemeInst.address,
                           contributionRewardInst.address];
-      const paramsArray = [schemeRegisterParams, schemeGCRegisterParams, schemeUpgradeParams,contributionRewardParams];
+      const paramsArray = [NULL_HASH, NULL_HASH, NULL_HASH, NULL_HASH];
```

Comment:
This is now unnecessary, let's remove this from the controller and the rest of Arc.